tendermint/starport
# feat(types): create a small template to test `ValidateBasic` method

### What does this MR does?

Create test templates for the `ValidateBasic` methods

### How to test?

- Scaffold a chain and a type:
```shell
$ starport scaffold chain github.com/cosmonaut/test
$ starport s map map-example first second third:uint --index idOne,idTwo:uint,idThree
```

- Check the file `x/test/types/tx_map_test_test.go` and run the tests:
```shell
$ go test ./x/test/types/...
```

### Additional Considerations

Also, I found a bug when we scaffold a packet. The args index number are wrong. You can check trying:
```shell
$ starport s packet bar text --module consuming --ack foo:string,bar:int,foobar:bool
```
